configuration/src/main/java/io/airlift/configuration/ConfigBinder.java

```diff
@@ -110,6 +110,15 @@ private ConfigBinder(Binder binder)
         createConfigDefaultsBinder(key).addBinding().toInstance(new ConfigDefaultsHolder<>(key, configDefaults));
     }
 
+    /**
+     * Binds default values for all the instances of given config class for the current binder
+     */
+    public <T> void bindConfigGeneralDefaults(Class<T> configClass, ConfigDefaults<T> configDefaults)
```

Comment:
How is this different from `public <T> void bindConfigDefaults(Class<T> configClass, ConfigDefaults<T> configDefaults)`?